packages/salesforcedx-vscode-lightning/test/deprecatedClassName.test.ts

```diff
@@ -13,12 +13,13 @@ describe('SLDS Deprecated Class Name', () => {
   let res: vscode.Uri[];
 
   before(async () => {
-
     if (vscode.workspace.rootPath) {
-      res = await vscode.workspace.findFiles(path.join('**', '*.cmp'));
-      await vscode.workspace.openTextDocument(res[0]).then(
-        document => vscode.window.showTextDocument(document)
+      res = await vscode.workspace.findFiles(
+        path.join('**', 'DemoComponent.cmp')
```

Comment:
You should be precise and look exactly for the .cmp you know has the deprecatedclassnames.